derive/src/to_client.rs

```diff
@@ -134,6 +132,7 @@ fn generate_client_methods(methods: &[MethodRegistration]) -> Result<Vec<syn::Im
 				);
 				client_methods.push(client_method);
 			}
+			MethodRegistration::Notification { .. } => continue,
```

Comment:
Users may wonder why client notification methods are not generated. Not sure of the best approach here - we used to [print a message during compilation](https://github.com/paritytech/jsonrpc/pull/436/files#diff-77384f93cc36bb6db93182b47a771715L115) for pubsub but that was annoying. Alternatively could generate the method with `unimplemented!()`?

Also would be good to have an issue "Implement client support for notifications"

Reply:
I like the `unimplemented!()` idea, if that's not too much effort to get in.

Reply:
https://github.com/paritytech/jsonrpc/issues/461

Reply:
Excellent idea! I'll add that in.